umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2/data_contact.clj

```diff
@@ -203,10 +207,13 @@
 
 (defn- get-collection-contact-persons-and-groups
  "Get contact persons and contact groups not associated with a data center."
- [contacts]
- (let [non-data-center-contacts (filter #(nil? (:DataCenter %)) contacts)
+ [contacts data-centers]
+ (let [data-centers (distinct (map #(select-keys % [:ShortName :LongName]) data-centers))
+       non-data-center-contacts (filter #(or (nil? (:DataCenter %))
+                                             (not (contains? (set data-centers) (:DataCenter %))))
+                                        contacts)
        groups (group-by :Type non-data-center-contacts)]
-   {:ContactPersons (map :Contact (get groups :contact-person))
+   {:ContactPersons (map #(assoc % :Roles ["Technical Contact"]) (map :Contact (get groups :contact-person)))
```

Comment:
Yep made the updates. Thanks for pointing that out. I had forgotten about that part.